src/Extensions/Shopware/Plugin/Services/Install.php

```diff
@@ -57,6 +64,26 @@ public function install(Plugin $plugin, $shopwarePath, $inputActivate = false, $
     }
 
     /**
+     * @param $plugin
+     * @param $shopwarePath
+     */
+    public function checkForNewStructure($plugin, $shopwarePath)
+    {
+        $path = $shopwarePath . '/engine/Shopware/Plugins/Local/';
+        $pluginName = $plugin->name;
+
+        $destPath = $plugin->module . "/" . $pluginName;
+
+        $absPath = $path . '/' . $destPath;
+
+        if (file_exists($absPath . "/Bootstrap.php")) {
+            return;
+        }
+
+        rename($absPath, $shopwarePath . "/custom/plugins/" . $pluginName);
```

Comment:
Please use the apostroph instead of quotation marks. There are several occurrences in this PR.